docs/src/design/stack/crypto_ops.md

```diff
@@ -266,3 +266,35 @@ The `HORNEREXT` makes one memory access request:
 $$
 u_{mem} = \alpha_0 + \alpha_1 \cdot op_{mem\_readword} + \alpha_2 \cdot ctx + \alpha_3 \cdot s_{13} + \alpha_4 \cdot clk + \alpha_{5} \cdot h_{0} + \alpha_{6} \cdot h_{1} + \alpha_{7} \cdot h_{3} + \alpha_{8} \cdot h_{4}
 $$
+
+## ARITHMETICCIRCUITEVAL
+
+The `ARITHMETICCIRCUITEVAL` operation evaluates an arithmetic circuit, given its circuit description and a set of input values, using the [ACE](../chiplets/ace.md) chiplet and asserts that the evaluation is equal to zero.
```

Comment:
Maybe not for this PR, but I think `ARITHMETICCIRCUITEVAL` is a bit too long. Just `CIRCUITEVAL` would have probably worked just as well.

Reply:
Agreed. Created an issue for this #2036